src/util.js

```diff
@@ -0,0 +1,37 @@
+export const getAllMovies = () => {
+return fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')  
+  .then(response => { 
+      console.log(response.status)                  
+      if(!response.ok) {
```

Comment:
I'm not sure if pinpointing a 500 status code in this way is absolutely necessary, but it works for now